opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -419,4 +436,5 @@ class _OTelSDKConfigurator(_BaseConfigurator):
     """
 
     def _configure(self, **kwargs):
-        _initialize_components(kwargs.get("auto_instrumentation_version"))
+        # _initialize_components(kwargs.get("auto_instrumentation_version"))
+        _initialize_components(**kwargs)
```

Comment:
I think we covered this a bit in the SIG. It would be great to have this behavior be configurable with custom implementations of the distro. So instead of override a private `_initialize_components` function, we expose this through `configure` so distros can have dictate their own behavior. It also makes sense then to use `kwargs` in this case.

Reply:
When you say "expose it" do you mean have _initialize_componenets take kwargs as propose, or to expose _the function itself_ so a distro could inject a custom function instead of just overriding _configure_ with it's replacement?